src/gcovr/formats/html/write.py

```diff
@@ -95,14 +81,11 @@ def get_theme_color(html_theme: str) -> str:
     return html_theme.split(".")[1] if "." in html_theme else html_theme
 
 
-# Loading Jinja and preparing the environment is fairly costly.
-# Only do this work if templates are actually used.
-# This speeds up text and XML output.
-@Lazy
-def templates(options):
+@functools.lru_cache(maxsize=1)
+def templates(options: Options) -> Environment:
```

Comment:
I had problems with typing annotations and the Lazy class but the comment is outdated. Since the refactoring in #755 the lazy loading isn't needed anymore. The file itself is already imported lazy. If no HTML report is used the file isn't imported.